### DISCUSSION TITLE
Windows fatal exception: access violation when running github actions on windows latest

### DISCUSSION THREAD

**Discussion by gilchn52** [community/community#43684] [Projects and Issues]
### Select Topic Area

Bug

### Body

hi everyone, 
i've came across a bug which i hope you can help me understand or solve.
i've create a github actions pipeline , which is based on windows .
there is a step of testing the code , with pytest.
the problem : all the files permission on tests folder changed to rwx/r/r , which causes the above error ("Windows fatal...")
now , if i clone the repo with ubuntu , i see the permissions are rwx for everyone and test pass with flying colors , but when the github actions runs it , it changes them
to rw/r/r, which is odd.  

appreciate any help .

**Comment by jviall** [comment]
Same issue 3 years later!
